skypy/pipeline/_config.py

```diff
@@ -0,0 +1,55 @@
+import builtins
+from importlib import import_module
+import yaml
+
+__all__ = [
+    'skypy_config',
```

Comment:
I found the name confusing because it's specifically just an encapsulation of our YAML extensions. What do you think about calling it thus?

```suggestion
    'load_skypy_yaml',
```